core/src/main/java/com/dtolabs/rundeck/core/audit/AuditEvent.java

```diff
@@ -0,0 +1,84 @@
+package com.dtolabs.rundeck.core.audit;
+
+
+import com.dtolabs.rundeck.core.common.Framework;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Describes an audit event generated by the system.
+ *
+ */
+public interface AuditEvent extends Serializable {
+
+
+  /**
+   *
+   * @return The system framework interface..
+   */
+  Framework getFramework();
```

Comment:
If the framework is necessary as part of the event data, use the `IFramework` interface in place of `Framework`